### PR TITLE
translating modulo 0 - play of pattern

### PR DESCRIPTION
**Pull request by stefi-says**
## Summary of changes

_Description of the changes included in this pull request. 
Steps to follow to confirm the feature is completed or the bug/issue is resolved. 
Are there any dependencies? 
Does this code break any existing functionality or introduce any risk?_

## Relevant Context

- Feature: [ticket number](link to project management ticket)
- Relevant Pull Request: # 

## Further comments

--- 
## Types of changes

- [ ] Content edit, File upload, content addition
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Technical Documentation Update
- [ ] Other (please describe):

## Does this introduce a breaking change?

_Breaking change is defined as_ **"A fix or feature that would cause existing functionality to change."**

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added/changed necessary documentation (if appropriate)
- [ ] All new and existing tests passed.

### CODE REVIEW

File: content/pr/learn/module-0/play-of-pattern.mdx [hunk 1]
```diff
@@ -1,100 +1,103 @@
...
 
-> 💡 How can you respond most honestly, accurately, and lovingly?
+> 💡 Como você pode responder com mais honestidade, precisão e amor?
 
 <AudioPlayer src="https://kernel-audio.s3.amazonaws.com/M0E1+-+The+Play+of+Pattern.mp3"/>
 
 [![Contradictions - there's jazz going on!](/images/mod0-img/jazz_pattern.png)](https://drive.google.com/drive/folders/1-PURd0Re8wx7Kq6f2LtcEz0iC2X7Ry4t)
 
-#### Divining Our Humanity
+#### Decifrando Nossa Humanidade
```
Comment (gabrielfior) [inline content/pr/learn/module-0/play-of-pattern.mdx:81]: I would say "Profetizando" or "Endeusando"

File: content/pr/learn/module-0/play-of-pattern.mdx [hunk 2]
```diff
@@ -1,100 +1,103 @@
...
+### A Pergunta Honesta
 
-Those who have **positively** changed the world did so because they learnt how to negotiate complexity, rather than impose their own will on things. They answered their own questions as honestly and directly as they could.
+Aqueles que mudaram positivamente o mundo o fizeram porque aprenderam a negociar a complexidade em vez de impor sua própria vontade sobre as coisas. Eles responderam às suas próprias perguntas da maneira mais honesta e direta possível.
 
-Playing with pattern, and becoming conscious of the constant interplay of complementary opposites as a means of cultivating presence, leads us to one simple query:
+Brincar com o padrão e tornar-se consciente da interação constante de opostos complementares como meio de cultivar a presença nos leva a uma pergunta simples:
 
-> 💡 What is the question to which you are the universe's answer?
+> 💡 Qual é a pergunta no universo para qual a resposta é você?
```
Comment (gabrielfior) [inline content/pr/learn/module-0/play-of-pattern.mdx:71]: Suggestion - "Para qual pergunta o universo diria que a resposta é você?"

File: content/pr/learn/module-0/play-of-pattern.mdx [hunk 3]
```diff
@@ -1,100 +1,103 @@
...
+
 
-If you want to change the world, the world will inevitably act back and change you. The results of Web 2 applications have made this clear. The greater our collective humility, the more gentle this exchange will be as Web 3 comes to life.
+Se você quiser mudar o mundo, o mundo inevitavelmente reagirá e mudará você. Os resultados dos aplicativos da Web 2 deixaram isso claro. Quanto maior for nossa humildade coletiva, mais gentil será essa troca para a  Web 3.
 
-The critical idea here is that thinking consciously about trade-offs, and developing the ability to hold many different probabilities in mind simultaneously without allowing personal bias to obscure your view of all possible futures, allows you to pick The Middle Way more often than not. 
+A ideia principal aqui é pensar conscientemente sobre os prós e contras e desenvolver a capacidade de manter muitas probabilidades diferentes em mente,simultaneamente, sem permitir que vieses pessoais obscureçam sua visão de todos os resultados possíveis, permitindo assim que você escolha o Caminho do Meio com mais frequência.
```
Comment (gabrielfior) [inline content/pr/learn/module-0/play-of-pattern.mdx:50]: Suggestion -> "[...] permitindo assim que você escolha um meio termo com alguma frequência."

File: content/pr/learn/module-0/play-of-pattern.mdx [hunk 4]
```diff
@@ -1,100 +1,103 @@
...
 
 </StatusBanner>
 
-This is because there are no simple, one-size-fits-all answers to the questions we're interested in learning about and teaching. That is what makes them good questions.
+Isso ocorre porque não há respostas simples e únicas para as perguntas que estamos interessados em aprender e ensinar. Isso é o que as torna boas perguntas.
 
-**Our intention is to give you the framework to begin thinking in transformative ways about the most important questions raised by this Web 3 world we're building together.**
 
-Kernel Fellows will distinguish themselves by their ability to **identify patterns**. 
+**Nossa intenção é te fornecer  a estrutura para começar a pensar de maneira transformadora sobre as questões mais importantes levantadas pelo mundo da Web 3.**
```
Comment (gabrielfior) [inline content/pr/learn/module-0/play-of-pattern.mdx:19]: Suggestion -> [...] pelo mundo da Web 3 que esstamos construindo juntos.

**Comment by andytudhope** [comment]
Closing this for lack of activity. Thank you both and I made sure to implement your suggestions in Pedro's translation Gabriel :pray:

**Pull request closed without merging**